<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterData</class>
 <widget class="QWidget" name="MasterData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pb_armies">
     <property name="text">
      <string>Armies</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="bb_okcanc">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_units">
     <property name="text">
      <string>Units</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_texts">
     <property name="text">
      <string>Texts</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTableView" name="tv_master_data"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pb_models">
     <property name="text">
      <string>Models</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
